modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -102,13 +142,101 @@ spec:
 <1> Inject the images referenced by the Operator by using environment variables.
 <2> Specify each image by a digest (SHA), not by an image tag.
 <3> Also reference the Operator container image by a digest (SHA), not by an image tag.
+<4> The `periodSeconds` value must be higher than the `timeoutSeconds` value when configuring probes. If left unspecified, the `timeoutSeconds` default value is `1`. The `periodSeconds` default value is `10`.
+
 --
+=====
+====
+
+. Set an environment variable, such as `RELATED_IMAGE_MEMCACHED`, for the additional images used by the Operator in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached:1.4.36-alpine"
```

Comment:
@ryantking If we call `make bundle` in step 2, should this be the tag or the digest?

Reply:
The "make bundle USE_IMAGE_DIGESTS=true" step should be the last step.